src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -433,6 +433,22 @@ public static void handleBenefitType(SscsCaseData caseData) {
     public static void handleIbcaCase(SscsCaseData caseData) {
         caseData.getAppeal().getHearingOptions().setHearingRoute(LIST_ASSIST);
         caseData.getAppeal().getMrnDetails().setDwpIssuingOffice("IBCA");
+        RegionalProcessingCenter regionalProcessingCenter = caseData.getRegionalProcessingCenter();
+        RegionalProcessingCenter listAssistRegionalProcessingCenter = RegionalProcessingCenter.builder()
+            .faxNumber(regionalProcessingCenter.getFaxNumber())
+            .address4(regionalProcessingCenter.getAddress4())
+            .phoneNumber(regionalProcessingCenter.getPhoneNumber())
+            .name(regionalProcessingCenter.getName())
+            .address1(regionalProcessingCenter.getAddress1())
+            .address2(regionalProcessingCenter.getAddress2())
+            .address3(regionalProcessingCenter.getAddress3())
+            .postcode(regionalProcessingCenter.getPostcode())
+            .city(regionalProcessingCenter.getCity())
+            .email(regionalProcessingCenter.getEmail())
+            .hearingRoute(LIST_ASSIST)
+            .epimsId(regionalProcessingCenter.getEpimsId())
+            .build();
+        caseData.setRegionalProcessingCenter(listAssistRegionalProcessingCenter);
```

Comment:
can't we just set hearingRoute to LIST_ASSIST in regionalProcessingCenter rather than rebuilding it

Reply:
RegionalProcessingCenter class doesn't allow for setter's as its defined with @Value rather than @Data, didn't want to mess with existing stuff too much.